src/Cemu/FileCache/FileCache.cpp

```diff
@@ -287,7 +287,10 @@ uint8* _fileCache_compressFileData(const uint8* fileData, uint32 fileSize, sint3
 	Bytef* compressedData = (Bytef*)malloc(4 + compressedLen);
 	int zret = compress2(compressedData + 4, &compressedLen, uncompressedInput, uncompressedLen, 4); // level 4 has good compression to performance ratio
 	if (zret != Z_OK)
+	{
+		delete compressedData;
```

Comment:
```suggestion
		free(compressedData);
```
Delete is wrong here as the memory is allocated with `malloc`